helm-chart/openj9-jitserver-chart/enable-jitserver.md

```diff
@@ -0,0 +1,171 @@
+<!--
+Copyright (c) 2020, 2020 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Quick Start: Enabling JITServer Technology
+
+This quick start guide walks you through an example of enabling JITServer Technology for your java application. This guide uses [Open Liberty WebServer](https://openliberty.io) as a Java application. JITServer technology can be used for any Java application running on recent releases of OpenJ9 for Java 8 and Java 11. Please make sure that JITServer image is compatible with the JDK that runs your application, as described [here](./README.md#a-verify-the-application-jdk-version-and-select-the-appropriate-jitserver-image).
```

Comment:
There was another comment about updating Open Liberty WebServer to a more correct name.  I think it applies here as well